aip/client-libraries/4222.md

```diff
@@ -13,7 +13,103 @@ routing.
 
 ## Guidance
 
-Code generators **should** look at URI-based variables declared in the
+Code generators **should** use the annotations to generate client libraries that,
+on a per-rpc basis extract routing information from the request payload and
+add that information to the routing header.
+
+There are two annotations that specify how to extract routing information from
+the request payload: the [`google.api.routing`][routing] that specifies how to
```

Comment:
let's make a markdown list here:

- `google.api.routing` that ...
- `google.api.http` that ...

Reply:
:+1: